marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -366,7 +376,7 @@ analysisResultPane state =
                   ]
               ]
           ]
-      _ -> text "Analysing..."
+      Loading -> text ""
```

Comment:
No loading wheel or nuttin'?

Reply:
we changed the button to say `Analysing...` so this would be redundant and I think wouldn't look great.

Reply:
Ah, fair enough. 👍